test-cases/gutenberg/group.md

```diff
@@ -0,0 +1,333 @@
+# Group Block - Test Cases
+
+--------------------------------------------------------------------------------
+
+##### TC001
+
+### AppenderButton is render properly
+
+-   Add a `Group` block
+-   Check if `AppenderButton` rendered
+-   Add another block inside `Group` and deselect
+-   Check if `AppenderButton` is align with the content
+
+Expected look:  
+![AppenderAlignment](../resources/group-appender-alignment.png)
+
+--------------------------------------------------------------------------------
+
+##### TC002
+
+### DefaultAppender allows to open Inserter and add block outside the group
+
+-   Add a `Group` block
+-   Having `Group` block selected press on `DefaultAppender`
+-   Check if the block is added outside after the `Group`
+
+Expected look:  
+![SeparatorAfter](../resources/group-outer-separator-after.png)
+
+-   Having `Group` block selected long press on `DefaultAppender`
+-   Select where you want to add block after or before
+-   Check if the block is added outside after/before the `Group`
+
+Expected look:  
+![SeparatorBefore](../resources/group-outer-separator-before.png)
+
+--------------------------------------------------------------------------------
+
+##### TC003
+
+### Deep nesting is possible (iOS only)
+
+-   Add a `Group` block
+-   Nest blocks inside multiple `Group` blocks (at least 3 levels deep)
+-   Check if app do not crashes
+
+--------------------------------------------------------------------------------
+
+##### TC004
+
+### Check if Group placeholder is render on RootList
+
+-   Add a `Group` block
+-   Deselect
+-   Check if you are able to see `Group` placeholder
+
+Expected look:  
+![Progress](../resources/group-placeholder-root.png)
+
+--------------------------------------------------------------------------------
+
+##### TC005
+
+### Check if Group placeholder is render in nested structure
+
+-   Add a `Group` block
+-   Nest some `Group` inside
+-   Select top-most `Group` block in hierarchy
+-   Check if you are able to see `Group` placeholder of nested blocks wrapped with dashed border
+-   Go down in the hierarchy observing if margins is keept (the whole UI should remain the same)
+-   After selecting last `Group` you should be able to see it's `AppenderButton`
+
+Expected look:  
+![PlaceholderNestes](../resources/group-placeholder-nest-1.png)
+  
+![PlaceholderDeepNested](../resources/group-placeholder-nest-deep.png)
+
```

Comment:
These screenshots show the same thing, could you check?

Reply:
if this is how it is supposed to work let's put better descriptions under the images or use markers on the images to show what we mean

Reply:
The images is very similar but not the same, the differences are:
1. Floating toolbar breadcrumbs shows different hierarchy
2. The title is dimmed on one image because we go with the selection on deeper level

The behaviour at this moment should looks like that. In near future we will add some indicator that inform user about changing selection ( in this case when empty nested group it's hard to notice that selection changes because the UI doesn't change)

Anyway I have added the descriptions under the images to point the differences